02-cpu-configure/ioc-restart.c

```diff
@@ -0,0 +1,83 @@
+/* BPM IOCs Utils
+ * Copyright (C) 2024 CNPEM
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <string.h>
+
+/*
+ * Slot allowlists to sanitize user input
+ */
+const char* const afc_allowlist[] = {
+    "1", "2", "2-1", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12"
+};
+
+const char* const rffe_allowlist[] = {
+    "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "23"
+};
+
+/*
+ * Check if the user provided slot string is valid
+ */
+int sanitize_slot(const char* slot, const char* const allowlist[], size_t allowlist_size) {
+    for (size_t i = 0; i < allowlist_size; i++) {
+        if (strcmp(slot, allowlist[i]) == 0) {
+            return 0;
+        }
+    }
+    return -1;
+}
```

Comment:
This is 100% a nit, but I like it for building good habits. These should all be `static` (the allowlists and the function)